doc/devel/coding_style_guideline.rst

```diff
@@ -44,36 +44,36 @@ and readability across the library::
   import numpy.testing as npt
   import scipy as sp
 
-No alias should be used for `h5py`::
+No alias should be used for ``h5py``::
 
   import h5py
 
 -------------------
 Cython coding style
 -------------------
 DIPY recommends the use of the standard Python
-PEP8_ style when writing `Cython <https://cython.org/>` code.
+PEP8_ style when writing `Cython <https://cython.org/>`_ code.
 
 Cython-specific syntax should follow these additional rules:
 
 Imports
 -------
-The `cimport`s should add the `c` prefix to the usual Python import package
+The ``cimport`s`` should add the ``c`` prefix to the usual Python import package
```

Comment:
@areeshatariq Sorry, I failed to spot this one: the first one should be (double_backticks)cimport(double_backticks)s

Reply:
@areeshatariq, can you address those little comments? It is almost ready to go. Thank you!

Reply:
Having a look at the rendered file, it still does not look good. My bad for not having checked the proposal before. So this will work: (double_backticks)cimport(double_backticks)'s  (note the straight tick before the s).